Server docs (#18)
Knotx/knotx-server-http: assembler/README.md

# Knot.x Assembler Handler
This module contains [Handler](https://vertx.io/docs/apidocs/io/vertx/core/Handler.html)

> that joins all [`Fragments`](https://github.com/Knotx/knotx-fragment-api#Fragmrnt) and

Reviewer: ```suggestion
that joins all [`Fragments`](https://github.com/Knotx/knotx-fragment-api) and
```

> saves the result into the [`ClientRequest`](api/docs/asciidoc/dataobjects.adoc#clientresponse) body.

Reviewer: ```suggestion
saves the result into the [`ClientResponse`](api/docs/asciidoc/dataobjects.adoc#clientresponse) body.
```

## How does it work?
Fragment Assembler reads Fragments from the [`RequestEvent`](https://github.com/Knotx/knotx-server-http/blob/master/api/src/main/java/io/knotx/server/api/context/RequestEvent.java)
and joins them all into one string.

### How Fragments are being joined?

> Lets explain process of fragments join using example. Fragment Assembler reads [`ClientRequest`](api/docs/asciidoc/dataobjects.adoc#clientresponse)

Reviewer: ```suggestion
Lets explain the process of joining fragments using an example. Fragment Assembler reads [`ClientRequest`](api/docs/asciidoc/dataobjects.adoc#clientresponse)
```
